gtk-sys/Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [package]
 name = "gtk-sys"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
Strange version

Reply:
Was broken in https://github.com/gtk-rs/sys/commit/8b30f8692b82bbc07273460c21c9157897dcdc82

Reply:
Excellent catch! Fixed!